silabs-multiprotocol/translations/en.yaml

```diff
@@ -25,15 +25,22 @@ configuration:
     description: Enable tracing for the Co-Processor Communication daemon.
   otbr_enable:
     name: Enable OpenThread Border Router
-    description: Enable OpenThread Border Router agent.
+    description: Enable the OpenThread Border Router.
   otbr_log_level:
     name: OpenThread Border Router agent log level
     description: >-
       Set logging level of the OpenThread Border Router agent (otbr-agent).
+  otbr_infra_if:
+    name: OpenThread Border Router infrastructure interface
+    description: >-
+      HA host network interface name to bind the "infrastructure" side (vs the
+      "Thread" side) of the OpenThread Border Router to.  By default, the
+      first interface with an associated default route is used.
```

Comment:
We use what is considered primary by NetworkManager. Is the first interface with an associated default what NetworkManager considers primary? :thinking:

Reply:
I said "first" here because we used this code: `bashio::api.supervisor 'GET' '/network/info' '' 'first(.interfaces[] | select (.primary == true)) .interface
'`

However, I think (but am not entirely sure) that NetworkManager will only ever have one primary interface, and I think that is the most recently activated interface that has an associated default route (either IPv4 or IPv6 default route).